hug/interface.py

```diff
@@ -352,14 +352,19 @@ def __call__(self, *args, **kwargs):
             self._rewrite_params(kwargs)
         try:
             result = self.interface(**kwargs)
-            self.cleanup_parameters(kwargs)
-            self.api.delete_context(context)
+            if self.transform:
+                if hasattr(self.transform, 'context'):
+                    self.transform.context = context
+                if hasattr(self.transform, 'dump'):
```

Comment:
Do we need to support transform objects with dump method? It was removed in #661, I can also remove it in my PR.